joern-cli/frontends/rubysrc2cpg/src/test/scala/io/joern/rubysrc2cpg/dataflow/DataFlowTests.scala

```diff
@@ -2246,4 +2246,35 @@ class DataFlowTests extends RubyCode2CpgFixture(withPostProcessing = true, withD
     }
   }
 
+  "Data flow through a keyword" should {
```

Comment:
Nit, but it's not through a keyword - it's a regular (keyword?-named) method

Reply:
Done